src/assets/toolkit/styles/utils/text.css

```diff
@@ -16,6 +16,16 @@
   font-weight: var(--font-weight-semibold) !important;
 }
 
+
+/**
+ * Text color
+ */
+
+.u-textWhite {
+  color: var(--color-white);
+}
+
```

Comment:
Does that mean this text is now inconsistent with other instances of header text?

If we need to adjust contrast for _all_ the headers, I'd rather do that.